### PR TITLE
feat: Dynamic error percentage

### LINKED ISSUE #109: FRP: Dynamic error percentage
### What is your proposal 🚀

Follow up on https://github.com/yearn/ySync/issues/105, make it so that the error percentage is only relative to the selected filters and not for all items.

ySync

### What does this solve ? 🛠

Easier to see the % of errors for each item(s)

### PR DESCRIPTION
**Pull request by karelianpie**
## Description

Only consider the error if that item is selected

## Related Issue

Closes https://github.com/yearn/ySync/issues/109

## Motivation and Context

Easier to see the % of errors for each item(s)

## How Has This Been Tested?

Locally, you can click on the items you're interested in and observe the error ratio changing accordingly.

### No errors

Screenshot 2023-08-08 at 13 36 34

### 100% errors (translations, don't panic 😅 )

Screenshot 2023-08-08 at 13 36 25

### Some errors

Screenshot 2023-08-08 at 13 36 31

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -211,23 +211,67 @@ function Index(): ReactNode {
...
+						(appSettings.shouldShowStrategies && vault.strategies.length === 0)
+						|| (appSettings.shouldShowMissingTranslations && !!vaultData.missingTranslations)
+						|| (appSettings.shouldShowIcons && !vaultData.hasValidIcon)
+						|| (appSettings.shouldShowIcons && !vaultData.hasValidTokenIcon)
+						|| (appSettings.shouldShowPrice && !vaultData.hasValidPrice)
+						|| (appSettings.shouldShowRetirement && !vaultData.hasValidRetirement)
+						|| (appSettings.shouldShowYearnMetaFile && !vaultData.hasYearnMetaFile)
+						|| (appSettings.shouldShowLedgerLive && !vaultData.hasLedgerIntegration.deployed)
+						|| (appSettings.shouldShowStrategies && !vaultData.hasValidStrategiesDescriptions) // TODO: check if this is correct
+						|| (appSettings.shouldShowStrategies && !vaultData.hasValidStrategiesRisk) // TODO: check if this is correct
```
Comment (karelianpie) [inline pages/index.tsx:317]: @Majorfi do you know what these are?

Comment (Majorfi) [reply pages/index.tsx:317]: wdym?
It's to check if risk score is not 0 or if the meta file has a description

**Review by Majorfi — APPROVED** [review]

File: pages/index.tsx (hunk 1 above)
Comment (karelianpie) [reply pages/index.tsx:317]: They don't have their `AnomaliesSection`, so are they even being used anywhere?

Comment (Majorfi) [reply pages/index.tsx:317]: It's supposed to be in the `Want Token Description` group or in the `Risk` group (checkbox to filter)

Comment (karelianpie) [reply pages/index.tsx:317]: > It's supposed to be in the `Want Token Description` group or in the `Risk` group (checkbox to filter)

Hmm `hasValidStrategiesDescriptions` should be in the `Want Token Description`? I don't get it

**Comment by 0xBasically** [comment]
Will check today

**Comment by 0xBasically** [comment]
Doesnt seem to work with 'Price' checkbox

**Review by 0xBasically — APPROVED** [review]
lgtm

File: pages/index.tsx (hunk 1 above)
Comment (karelianpie) [reply pages/index.tsx:317]: Discussed with Major, let's address this later. AFAIK these are not being used, so safe to ignore for now.

**Pull request merged**